### ISSUE TITLE
Website should be navigable for screen-reader

### ISSUE THREAD

**Issue by lascellesabercrombie** [lascellesabercrombie/turn2us-challenge#5]

**Comment by lascellesabercrombie** [comment]
Tested and working
